### PR TITLE
Java 20643

### PR DESCRIPTION
**Pull request by Bipinkumar27**

### CODE REVIEW

**Comment by dkapil** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -364,7 +364,7 @@
 
                 <module>muleesb</module>
 
-                <module>web-modules</module>
+                <!--<module>web-modules</module>-->
```
Comment (dkapil) [inline pom.xml:367]: instead of commenting, remove them

File: pom.xml [hunk 2]
```diff
@@ -1195,6 +1196,8 @@
                 <module>persistence-modules/questdb</module>
                 <module>vaadin</module>
                 <module>libraries-3</module>
+                <module>web-modules/apache-tapestry</module>
+                <!--<module>web-modules/apache-tapestry</module>-->
```
Comment (dkapil) [inline pom.xml:1200]: add the entire web-modules here and then in the pom.xml of web-modules enable those that. you have upgraded for e.g. apache-tapestry and comment out the rest.

**Review by dkapil — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (Bipinkumar27) [reply pom.xml:1200]: sure but I have divided the task into sub-tasks. Should I add all the rest modules in their jira commit?

File: pom.xml (hunk 1 above)
Comment (Bipinkumar27) [reply pom.xml:367]: sure

**Comment by dkapil** [comment]
test this please

**Pull request merged**